android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -125,9 +124,7 @@ function onStateChange(change) {
 // Query KMW if a given keyboard uses chiral modifiers.
 function setIsChiral(keyboardProperties) {
   var name = typeof(keyboardProperties.internalName) == 'undefined' ? keyboardProperties.keyboardName : keyboardProperties.internalName;
-  var kmw=window['keyman'];
-  var isChiral = kmw.isChiral(name);
-
+  var isChiral = keyman.isChiral(name);
   window.jsInterface.setIsChiral(isChiral);
   return true;
 }
```

Comment:
Is there a `keyman.` alternative for `KeymanWeb.registerStub` on line 133 below?

Reply:
@jahorton ?

Reply:
`keyman.interface` is a `keyman`-object based reference to the `KeymanWeb` keyboard-interfacing object.

So, `keyman.interface.registerStub` would technically work.

Note that `KeymanWeb` is guaranteed to exist b/c all KMW keyboards refer to it for rule processing.

Reply:
Thanks @jahorton. Agree `KeymanWeb.` is still going to be good. I like `keyman.interface.` as a more consistent approach internally here!